colossalai/pipeline/schedule/interleaved_pp.py

```diff
@@ -27,15 +28,25 @@ def __init__(
         assert (
             num_microbatch is not None or microbatch_size is not None
         ), "Either num_microbatch or microbatch_size should be provided"
+        self.logger = get_dist_logger()
+        self.logger.warning("If pipeline hangs, please enlarge NCCL_BUFFSIZE.")
```

Comment:
Can we enlarge `NCCL_BUFFSIZE` automatically?

Reply:
15fdb26 fix this.